src/pages/notificationDeepLink/index.tsx

```diff
@@ -15,7 +15,7 @@ export const NotificationDeepLink = () => {
     try {
       const res = await getNotificationApi(notificationId);
       setNotification(res);
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
any?

Reply:
the errors returned from API usually have different structures. I couldn't find a specific type. Do you have any better idea?

Reply:
it's better to create different types or custom classes (APIError, NetworkError, ...) and check types (Type Quard) but let's skip for now until BE has the same structure for every APIError